qabelbox/src/androidTest/java/de/qabel/qabelbox/helper/MockedBoxProviderTest.java

```diff
@@ -41,8 +43,8 @@ public void setUp() throws Exception {
     protected void configureTestServer() {
 
         new AppPreference(getContext())
-                .setToken(getContext().getString(R.string.blockserver_magic_testtoken));
-        URLs.setBaseBlockURL(getContext().getString(R.string.testBlockServer));
+                .setToken(getContext().getString(string.blockserver_magic_testtoken));
```

Comment:
I do not think this is a change we actually want to make. The R-import is pretty much accepted standard.